<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ResourceDialog</class>
 <widget class="QDialog" name="ResourceDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>451</width>
    <height>629</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Resource Properties</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout_2">
     <property name="labelAlignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <item row="0" column="1">
      <widget class="QLineEdit" name="leTitle">
       <property name="placeholderText">
        <string>标题</string>
       </property>
       <property name="clearButtonEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="leAlias">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="placeholderText">
        <string>别名</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="2">
      <widget class="QStackedWidget" name="ui_stwProperty">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="page">
        <layout class="QFormLayout" name="formLayout">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item row="0" column="0" colspan="2">
          <widget class="QPlainTextEdit" name="pteDesc">
           <property name="placeholderText">
            <string>描述</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="ui_imgPage">
        <layout class="QGridLayout" name="gridLayout">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item row="0" column="1">
          <widget class="QPushButton" name="ui_pbOpenImgFile">
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLineEdit" name="ui_leImgFile">
           <property name="text">
            <string/>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="0" colspan="2">
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>序号</string>
       </property>
       <property name="buddy">
        <cstring>spbSeq</cstring>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QSpinBox" name="spbSeq">
       <property name="minimum">
        <number>-1</number>
       </property>
       <property name="maximum">
        <number>9999</number>
       </property>
       <property name="value">
        <number>-1</number>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QCheckBox" name="ui_cbPrivate">
       <property name="text">
        <string>私有</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QCheckBox" name="ui_cbHide">
       <property name="text">
        <string>隐藏</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>标题</string>
       </property>
       <property name="buddy">
        <cstring>leTitle</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>别名</string>
       </property>
       <property name="buddy">
        <cstring>leAlias</cstring>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QRadioButton" name="rbArticle">
         <property name="text">
          <string>文章</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
         <property name="value" stdset="0">
          <string notr="true">A</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">bgType</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="rbComponent">
         <property name="text">
          <string>组件</string>
         </property>
         <property name="value" stdset="0">
          <string notr="true">C</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">bgType</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="rbStyle">
         <property name="text">
          <string>格式</string>
         </property>
         <property name="value" stdset="0">
          <string>S</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">bgType</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="rbMedia">
         <property name="text">
          <string>媒体</string>
         </property>
         <property name="value" stdset="0">
          <string notr="true">M</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">bgType</string>
         </attribute>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>类型</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>leTitle</tabstop>
  <tabstop>leAlias</tabstop>
  <tabstop>spbSeq</tabstop>
  <tabstop>ui_cbPrivate</tabstop>
  <tabstop>ui_cbHide</tabstop>
  <tabstop>rbArticle</tabstop>
  <tabstop>rbComponent</tabstop>
  <tabstop>rbStyle</tabstop>
  <tabstop>rbMedia</tabstop>
  <tabstop>pteDesc</tabstop>
  <tabstop>ui_leImgFile</tabstop>
  <tabstop>ui_pbOpenImgFile</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ResourceDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>275</x>
     <y>619</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ResourceDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>343</x>
     <y>619</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="bgType"/>
 </buttongroups>
</ui>
